srcd/superset/assets/src/customization/colors.js

```diff
@@ -2,23 +2,45 @@
 
 const rgba = color => `rgba(${color.r}, ${color.g}, ${color.b}, ${color.a})`;
 
-// from current superset configuration
-const mainColorSpecs = { r: 0, g: 122, b: 135, a: 1 };
+// source{d} royal (#8719cb)
+const mainColorSpecs = { r: 135, g: 25, b: 203, a: 1 };
 
-// from current superset configuration
 const palette = {
-  rausch: '#ff5a5f',
-  hackb: '#7b0051',
-  kazan: rgba(mainColorSpecs),
-  babu: '#00d1c1',
-  lima: '#8ce071',
-  beach: '#ffb400',
+  royal: "#7015a8",
+  royalMiddle: rgba(mainColorSpec),
+  royalDeep: "#591085",
+  royalLight: "#a438e7",
+  royalLightDeep: "#bc6ded",
+
+  lime: "#008168",
+  limeMiddle: "#009b7c",
+  blueDeep: "#006853",
+  limeLight: "#00b491",
+  blueLightDeep: "#00c8a1",
+
+  coral: "#e5770a",
+  coralMiddle: "#f68b0c",
+  coralDeep: "#d66600",
+  coralLight: "#f89c30",
+  coralLightDeep: "#f9ae56",
+
+  vanila: "#d024c6",
+  vanilaLight: "#e981e3",
+
+  navy: "#195dca",
+  navyLight: "#8fb5f1",
+
+  sky: "#29bff2",
+  skyLight: "#c9effc",
+
+  gray: "#6d6e71",
+  grayLight: "#c6c7c8"
 };
 
 const colors = {
   mainSpecs: mainColorSpecs,
   main: rgba(mainColorSpecs),
-  palette,
+  palette
```

Comment:
Needs to have the trailing comma
```suggestion
  palette,
```